prod/php/ElasticOTel/PhpPartFacade.php

```diff
@@ -272,6 +276,17 @@ private static function registerOtelLogWriter(): void
         ElasticLogWriter::enableLogWriter();
     }
 
+    private static function registerNativeOtlpSerializer(): void
+    {
+        if (elastic_otel_get_config_option_by_name('native_otlp_serializer_enabled') === false) {
+            BootstrapStageLogger::logDebug('ELASTIC_OTEL_NATIVE_OTLP_SERIALIZER_ENABLED set to false', __FILE__, __LINE__, __CLASS__, __FUNCTION__);
+        } else {
+            class_exists("Elastic\\OTel\\OtlpExporters\\SpanExporter");
```

Comment:
Are these `class_exists` calls to invoke autoload? Could you please clarify (maybe it would be worth adding a comment) why is it necessary?